packages/schemas/src/types/user.ts

```diff
@@ -15,11 +17,26 @@ export const userInfoSelectFields = Object.freeze([
   'isSuspended',
 ] as const);
 
-export type UserInfo<Keys extends keyof CreateUser = (typeof userInfoSelectFields)[number]> = Pick<
-  CreateUser,
+export type UserInfo<Keys extends keyof User = (typeof userInfoSelectFields)[number]> = Pick<
+  User,
   Keys
 >;
 
+export const userInfoResponseGuard: CreateGuard<UserInfo> = Users.guard.pick({
```

Comment:
I alse add this in https://github.com/logto-io/logto/pull/3809

Reply:
racing time！